jwala-services/src/main/java/com/cerner/jwala/service/jvm/impl/JvmServiceImpl.java

```diff
@@ -86,6 +103,9 @@
     private final HistoryFacadeService historyFacadeService;
     private final BinaryDistributionService binaryDistributionService;
     private final FileUtility fileUtility;
+    //JDK Upgrade Resource file names Array
+   private static String[] jdk_upgrade_resourceNamesArray= {"setenv"};

```

Comment:
Formatting: fix the spacing and the name of the array doesn't match Java standards

Reply:
This is fixed in the latest check-in.